backend-processing.ts

```diff
@@ -160,23 +151,40 @@ export default class BackendProcessing {
     if (mis.length == 1) mis = "0" + mi;
     return `${s[s.length - 1]}-${mis}-01`;
   }
-
-  async check_connection(): Promise<boolean> {
-    let con = await this.create_connection_to_database();
-    return new Promise<boolean>((resolve, reject) => {
-      con.query("SELECT count(*) FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'heroku_2396736b79200ba'", (err, rows) => {
-        if (err) {
-          this.end_connection(con);
-          reject(err);
-        }
-        this.end_connection(con);
-        resolve(true);
-      });
-    });
+  async insert_person(person){
```

Comment:
Thanks for the comments. We want the add/modify/delete methods to be async so that the backend can wait for it to finish, then tell the frontend if it was successful or not. This might be clearer if the functions returned a boolean, so we will rewrite it like that.